sdk/spring/tests.yml

```diff
@@ -1,9 +1,33 @@
 trigger: none
 
 jobs:
-  - template: ../../eng/pipelines/templates/jobs/archetype-sdk-tests.yml
-    parameters:
-      ServiceDirectory: spring
-      EnvVars:
-        AZURE_TEST_MODE: LIVE
-      Artifacts: []
+    - template: ../../eng/pipelines/templates/jobs/archetype-sdk-tests.yml
+      parameters:
+        TimeoutInMinutes: 240
+        MaxParallel: 2
+        ServiceDirectory: spring
+        TestStepMavenInputs:
+          options: '-Dmaven.wagon.http.pool=false $(DefaultOptions) -Dmaven.javadoc.skip=true -Drevapi.skip=true -DskipSpringITs=false -pl $(ProjectList)'
+          goals: "verify"
+
+        Artifacts:
+          - name: azure-spring-boot-tests
+            groupId: com.microsoft.azure
+            safeName: azurespringboot-tests
+          - name: azure-spring-boot-test-core
+            groupId: com.microsoft.azure
+            safeName: azurespringboottestcore
+          - name: azure-spring-boot-test-cosmosdb
+            groupId: com.microsoft.azure
+            safeName: azurespringboottestcosmosdb
+          - name: azure-spring-boot-test-aad
+            groupId: com.microsoft.azure
+            safeName: azurespringboottestaad
+          - name: azure-spring-boot-test-keyvault
+            groupId: com.microsoft.azure
+            safeName: azurespringboottestkeyvault
+          - name: azure-spring-boot-test-application
+            groupId: com.microsoft.azure
+            safeName: azurespringboottestapplication
+        EnvVars:
+          AZURE_TEST_MODE: LIVE
```

Comment:
I'm seeing [failures ](https://dev.azure.com/azure-sdk/internal/_build/results?buildId=422661&view=ms.vss-test-web.build-test-results-tab&runId=11435898&resultId=100001&paneView=debug) here that look like they're related to the right environment variables for an AAD app not being present. If you're using the default azure credentials then setting this in the `EnvVars` section might fix it: 

```
        ARM_CLIENTID: $(aad-azure-sdk-test-client-id)
        ARM_CLIENTKEY: $(aad-azure-sdk-test-client-secret)
        AZURE_TENANT_ID: $(aad-azure-sdk-test-tenant-id)
```

Reply:
@danieljurek We're using another Azure AD (which is created by ourselves) so we'll need to add such environment variables for the pipeline. Should we create a variable group for these variables or is there somewhere we could add them to? Seems like I don't have the permission to edit the [pipeline](https://dev.azure.com/azure-sdk/internal/_build?definitionId=1760&_a=summary).